### PR TITLE
http2: always enable grpc stats filter

### PR DESCRIPTION
**Pull request by zirain**
I think this should always on when protocol is grpc

### CODE REVIEW

File: internal/xds/filters/wellknow.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
```
Comment (arkodg) [inline internal/xds/filters/wellknow.go:5]: misspell - wellknown.go

**Comment by arkodg** [comment]
is this needed for https://github.com/envoyproxy/gateway/issues/699 ?

**Comment by zirain** [comment]
> is this needed for #699 ?

I recall this when I'm refactoring the monitoring pkg in istio repo.

File: internal/xds/filters/wellknow.go (hunk 1 above)
Comment (zirain) [reply internal/xds/filters/wellknow.go]: thanks, fixed

**Comment by arkodg** [comment]
> > is this needed for #699 ?
> 
> I recall this when I'm refactoring the monitoring pkg in istio repo.

looks like it will be needed for grpc stats https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/grpc_stats/v3/config.proto#envoy-v3-api-msg-extensions-filters-http-grpc-stats-v3-filterconfig

**Review by arkodg — APPROVED** [review]

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**